libimage/runtime.go

```diff
@@ -53,6 +53,11 @@ type Runtime struct {
 	systemContext types.SystemContext
 }
 
+// Returns a copy of the runtime's system context.
+func (r *Runtime) SystemContext() *types.SystemContext {
```

Comment:
nit:
```suggestion
func (r *Runtime) SystemContextCopy() *types.SystemContext {
```

Reply:
I had the same thought initially but prefer with `Copy`.  I think users don't need to worry but internal callers should be aware it's a copy.

Reply:
Sounds good :+1:

Reply:
Cool to drop a `/lgtm`?